### DISCUSSION TITLE
corrección plataforma curso IA

### DISCUSSION THREAD

**Discussion by rramosp** [rramosp/ai4eng.v1#127] [UDEA Students]
hola a tod@s, 

acabo de corregir las evaluaciones del curso en la plataforma para que corresponda con la estructura del curso que explicamos este martes pasado.

debéis de poder entrar de nuevo tanto en la plataforma como desde los notebooks.

observad que:
el curso y sesión se llaman ahora ai4eng.v1.udea.r2::2022.1 (mirad los pantallazos más abajo)
eliminé las inscripciones que teníais de versiones anteriores del curso

cualquier inquietud hacednos saber **en este foro**

saludos
raúl

![image](https://user-images.githubusercontent.com/13835425/169654592-502af0a5-c88a-4955-9723-d9751abaf31a.png)
![image](https://user-images.githubusercontent.com/13835425/169654596-e750baa6-53f6-4587-8f69-c1ea740cf763.png)

**Comment by valentinaechavarria** [comment]
Profesor buenos días, profesor me podría explicar por qué motivos puede suceder que al poner a correr las bibliotecas, estas nunca se ejecuten y entre en un estado de cola de más de 20 minutos, es que esto me está dificultando la realización de las actividades dado que si no me corre ese renglón, no puedo ni acceder a login.

**Reply by Felipe-RA** [reply]
Saludos valentina

¿Ya probaste a reiniciar el kernel?
¿Lo estas corriendo desde Google Colab?

También, para mantener las discusiones centradas en el tema que toca te agradecería si la próxima vez creas una nueva discusión para tu problema concreto

**Reply by valentinaechavarria** [reply]
Felipe perdón, es mi primera vez en esta plataforma, en una próxima oportunidad la crearé, sí, estaba intentado correrlo desde colab, hasta borré el cache, pero ya lo logré solucionar almacenando en GPU, pero no sé si esto es correcto.

**Comment by JuanJoseOspinar** [comment]
Buenos días profesor, no estoy en el grupo nuevo que creo, a ver si me puede mover al nuevo grupo porfa,juan.ospinar1@udea.edu.co.Muchas gracias
